### ISSUE TITLE
Actually Program may need more than default node memory to run.

### ISSUE THREAD

**Issue by 3vivekb** [brendannee/gtfs-to-html#27]
Currently when I run the program with my Agency's gtfs file it spits out Javascript heap out of memory errors.

**Comment by brendannee** [comment]
Can you link me to the GTFS file you are using so I can try it out?

**Comment by 3vivekb** [comment]
I was using the VTA gtfs from transitfeeds but it has many routes that don't have trips associated with them.  I started cleaning out as many routes that gave me errors.  It's possible there are also trips without routes.
https://github.com/3vivekb/vta_cleaned_gtfs

**Comment by 3vivekb** [comment]
I ran which gtfs-to-html and then ran:
'''node --max_old_space_size=2000 /usr/local/bin/gtfs-to-html'''
This allocates 2 gigs or ram for node over the default 512 megs.   It maxed out.  I think it's hitting a bad loop.

**Comment by brendannee** [comment]
Can you make sure you are running the latest version of gtfs-to-html and delete and reinstall node_modules? I just pushed some updates to the code that solve issues which should be unrelated to what you are experiencing.

I was able to run it successfully against the GTFS file you provided.

<img width="606" alt="screen shot 2016-12-15 at 12 49 53 pm" src="https://cloud.githubusercontent.com/assets/96217/21222972/07fd12f0-c2c5-11e6-8a56-3503c4085fef.png">

**Comment by 3vivekb** [comment]
Hey @brendannee I got it to work.  I deleted all the old modules and reinstalled - but it still crashed.  But then I ran it with 3-4 gigs of ram and it didn't crash from out of memory errors.  I think with the setting to include non-timepoints and maps it just needed more ram.  I changed the settings and  ran it with only timepoints it worked without the special settings.

It's possible I installed it incorrectly, used the wrong permissions

I'm not sure why my results were different than yours, and why the log says only 1 route was created.
